### PR TITLE
add doc for recover-cmd

### PR DESCRIPTION
**Pull request by FingerLeader**
Signed-off-by: FingerLeader <wanxfinger@gmail.com>

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-process-chaos-in-physical-nodes.md [hunk 1]
```diff
@@ -52,19 +52,22 @@ Usage:
...
+      --log-level string   the log level of chaosd. The value can be 'debug', 'info', 'warn' and 'error'
+      --uid string         the experiment ID
 ```
 
 #### 模拟进程被终止相关配置说明
 
 | 配置项 | 配置缩写 | 说明 | 值 |
 | :-- | :-- | :-- | :-- |
 | `process` | p | 需要注入故障的进程的名字或者进程的标识符 | string 类型，默认为 `""` |
+| `recover-cmd` | r | 需要在实验恢复时执行的命令 | | string 类型，默认为 `""`|
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-process-chaos-in-physical-nodes.md:70]:
```suggestion
| `recover-cmd` | r | 需要在实验恢复时执行的命令 | string 类型，默认为 `""`|
```

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 2]
```diff
@@ -52,19 +52,22 @@ Usage:
...
+      --log-level string   the log level of chaosd. The value can be 'debug', 'info', 'warn' and 'error'
+      --uid string         the experiment ID
 ```
 
 #### Configuration description for killing a process
 
 | Configuration item | Abbreviation | Description | Value |
 | :-- | :-- | :-- | :-- |
 | `process` | p | The name or the identifier of the process to be injected faults | string; the default value is `""`. |
+| `recover-cmd` | The command will be executed when recovering experiment | string; the default value is `""`. |
```
Comment (en-jin19) [inline docs/simulate-process-chaos-in-physical-nodes.md:70]:
```suggestion
| `recover-cmd` | r | The command to be run when recovering experiment | string; the default value is `""`. |
```

**Comment by FingerLeader** [comment]
PTAL /cc @WangXiangUSTC

**Review by WangXiangUSTC — APPROVED** [review]
please fix the ci, rest LGTM

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 3]
```diff
@@ -52,19 +52,22 @@ Usage:
...
+      --log-level string   the log level of chaosd. The value can be 'debug', 'info', 'warn' and 'error'
+      --uid string         the experiment ID
 ```
 
 #### Configuration description for killing a process
 
 | Configuration item | Abbreviation | Description | Value |
 | :-- | :-- | :-- | :-- |
 | `process` | p | The name or the identifier of the process to be injected faults | string; the default value is `""`. |
+| `recover-cmd` | The command to be run when recovering experiment | string; the default value is `""`. |
```
Comment (en-jin19) [inline docs/simulate-process-chaos-in-physical-nodes.md:70]:
```suggestion
| `recover-cmd`` | r |The command to be run when recovering experiment | string; the default value is `""`. |
```

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 4]
```diff
@@ -52,19 +52,22 @@ Usage:
   chaosd attack process kill [flags]
 
 Flags:
-  -h, --help             help for kill
-  -p, --process string   The process name or the process ID
-  -s, --signal int       The signal number to send (default 9)
+  -h, --help                 help for kill
+  -p, --process string       The process name or the process ID
+  -r, --recover-cmd string   The command to be executed when recovering experiment
```
Comment (en-jin19) [inline docs/simulate-process-chaos-in-physical-nodes.md:57]:
```suggestion
  -r, --recover-cmd string   The command to be run when recovering experiment
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-process-chaos-in-physical-nodes.md [hunk 5]
```diff
@@ -52,19 +52,22 @@ Usage:
   chaosd attack process kill [flags]
 
 Flags:
-  -h, --help             help for kill
-  -p, --process string   The process name or the process ID
-  -s, --signal int       The signal number to send (default 9)
+  -h, --help                 help for kill
+  -p, --process string       The process name or the process ID
+  -r, --recover-cmd string   The command to be executed when recovering experiment
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-process-chaos-in-physical-nodes.md:57]:
```suggestion
  -r, --recover-cmd string   The command to be run when recovering experiment
```

File: docs/simulate-process-chaos-in-physical-nodes.md (hunk 4 above)
Comment (en-jin19) [reply docs/simulate-process-chaos-in-physical-nodes.md:57]: 可以与 70 行保持一致

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**